src/main/java/tatracker/model/student/UniqueStudentList.java

```diff
@@ -120,6 +139,35 @@ public void setStudents(List<Student> students) {
         internalList.setAll(students);
     }
 
+    /**
+     * returns alphabetical comparator.
+     */
+    public Comparator<Student> getAlphabetically() {
+        return alphabetically;
+    }
```

Comment:
Oooh what's this used for?
I couldn't find another usage in the pr?

Reply:
I was getting check style problems for not having a getter... idk why haha